### PR TITLE
Move sscanf

### PR DESCRIPTION
**Pull request by levBagryansky**
@Graur , please review
I copied sscanf according to #9

### CODE REVIEW

File: src/main/eo/org/eolang/txt/sscanf.eo [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# The MIT License (MIT)
+#
+#  2022 Eugene Darashkevich
```
Comment (Graur) [inline src/main/eo/org/eolang/txt/sscanf.eo:3]: @levBagryansky I think here should be `#  2016-2022 Objectionary.com`

File: src/main/eo/org/eolang/txt/text.eo [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # The MIT License (MIT)
 #
-#  2016-2022 Objectionary.com
+#  2022 Eugene Darashkevich
```
Comment (Graur) [inline src/main/eo/org/eolang/txt/text.eo:3]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

File: src/main/java/EOorg/EOeolang/EOtxt/EOsscanf.java [hunk 3]
```diff
@@ -0,0 +1,371 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Eugene Darashkevich
```
Comment (Graur) [inline src/main/java/EOorg/EOeolang/EOtxt/EOsscanf.java:4]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

File: src/test/eo/org/eolang/txt/sscanf-tests.eo [hunk 4]
```diff
@@ -0,0 +1,165 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2022 Eugene Darashkevich
```
Comment (Graur) [inline src/test/eo/org/eolang/txt/sscanf-tests.eo:3]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

File: src/test/java/EOorg/EOeolang/EOtxt/EOsscanfTest.java [hunk 5]
```diff
@@ -0,0 +1,185 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Eugene Darashkevich
```
Comment (Graur) [inline src/test/java/EOorg/EOeolang/EOtxt/EOsscanfTest.java:4]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

**Review by Graur — CHANGES_REQUESTED** [review]
@levBagryansky Good one! Just change the license owner

**Review by Graur — APPROVED** [review]
@yegor256 please, have a look

**Comment by levBagryansky** [comment]
@yegor256 look please

**Comment by yegor256** [comment]
@rultor merge

**Comment by Graur** [comment]
@levBagryansky Thanks!

**Pull request merged**